setup.py

```diff
@@ -21,6 +21,7 @@
         "psutil",
         "deprecated",
         "attrs",
+        "janis-pipelines.core >= 0.11.2, <= 0.12.0",
```

Comment:
Out of interest, why the maximum version? If the API changes, should this not be using the latest API?

Reply:
I wasn't sure what you opinion of pinning to specific versions. Theoretically any version > = 0.11.2, I don't think we've changed the syntax of anything here in 18 of the past 24 months, so It's pretty safe. I've expanded the range.

Reply:
I want to avoid pinning versions in case aCLImatise is itself used as a library and needs to not be too restrictive.